src/api/nodeRpcCall.js

```diff
@@ -13,7 +13,7 @@ import { convertAssetData } from '../utils/dexFormatter';
 import { parseDollars, parseMerits } from '../utils/walletHelpers';
 import { getMetadataCache, setMetadataCache } from '../utils/nodeRpcCall';
 import { addReturns, calcInflation, getBaseInfo } from '../utils/staking';
-import identityJudgementEnums from "../constants/identityJudgementEnums";
+import identityJudgementEnums from '../constants/identityJudgementEnums';
```

Comment:
Not directly related to the PR, but pre-commit hook wasn't allowing me to commit without fixing this.